spec/unit/blockscore/person_spec.rb

```diff
@@ -1,10 +1,12 @@
 module BlockScore
   RSpec.describe Person do
     describe '.new' do
-      subject(:person) { BlockScore::Person.new(attributes_for(:person)) }
+      let(:attrs)      { attributes_for(:person) }
+      subject(:person) { BlockScore::Person.new(attrs) }
```

Comment:
Your sequence is contrary to the convention(?) in this code, and to my experience. I always put the lets first. @backus?